docs/source/workflows/performance_workflows.rst

```diff
@@ -87,7 +96,13 @@ For categorical responses, performance metrics include either the area under the
 -  Support-weighted F1 score is calculated from averaged precision and recall of the model, weighted by the in-class fraction of true positives according to the formula ``2.0 * precision * recall / (precision + recall) * fraction_true_positives`` summed over each class.
    Scores are bounded by 0 and 1. At a value of 1, the model has perfect precision and recall.
 
-The following demonstrates how to trigger workflow execution using an already existing `predictor` object and the `workflow` created in the example above.:
+In addition to the aforementioned metrics, predicted vs. actual data are also available.
+The structure of the data will depend on whether the response is numeric or categorical.
+For numeric responses, predicted and actual data contain the value and standard error associated with each data point.
+For categorical responses, class probabilities are returned.
+(If desired, standard error of categorical responses can be computed using the equation ``1 / max(class_probability) - 1`` or other desired metric.)
```

Comment:
```suggestion
(If desired, an precision-based metric for categorical responses can be computed using the equation ``1 / max(class_probability) - 1``.)
```

Reply:
I'm not sure what this line means.  If desired, the user can by themselves compute this?  Should we even be mentioning it (I don't think it stands on strong statistical footing)?  cc: @gregor-robinson

Reply:
Removed.